blank_project/src/main.js

```diff
@@ -5,6 +5,8 @@ import getConfig from "./config"
 
 let nearConfig = getConfig(process.env.NODE_ENV || "development");
 window.nearConfig = nearConfig;
+const nearGreetings = ['Hello', 'Aloha', 'Bonjour'];
+let nearGreetingIndex = 0;
```

Comment:
I think this comment still applies, right?

![image](https://user-images.githubusercontent.com/221614/85639905-34f3ed80-b658-11ea-9e84-cdea572a8eef.png)

Reply:
yes, in the same way that if someone refreshed create-near-app now the "change greeting" button would do nothing.
what would you like to see here? a map keeping track of what came back from the chain and updating the index?

Reply:
> in the same way that if someone refreshed create-near-app now the "change greeting" button would do nothing

great point